wait/host.go

```diff
@@ -158,18 +158,10 @@ func (a *HostAwaitility) WaitForDeletedMasterUserRecord(name string) error {
 	})
 }
 
-func getUaSpecSyncIndex(mur *toolchainv1alpha1.MasterUserRecord, targetCluster string) string {
-	for _, ua := range mur.Spec.UserAccounts {
-		if ua.TargetCluster == targetCluster {
-			return ua.SyncIndex
-		}
-	}
-	return ""
-}
-
-func containsUserAccountStatus(uaStatuses []toolchainv1alpha1.UserAccountStatusEmbedded, uaStatus toolchainv1alpha1.UserAccountStatusEmbedded) bool {
-	for _, status := range uaStatuses {
-		if reflect.DeepEqual(uaStatus, status) {
+func containsUserAccountStatus(expectedStatuses []toolchainv1alpha1.UserAccountStatusEmbedded, actualStatus toolchainv1alpha1.UserAccountStatusEmbedded) bool {
```

Comment:
isn't it the other way around - the first param contains the actual statuses and the second is the expected one?

Reply:
well, most of the time in the `stretch/testify` functions, the first arg is the expectation.

Reply:
I have two comments to that:
1. agree that most of the testify assertions/functions has the first param as expectation, but it doesn't apply for `assert.Contains` function - see:
```
// Contains asserts that the specified string, list(array, slice...) or map contains the
// specified substring or element.
//
//    assert.Contains(t, "Hello World", "World")
//    assert.Contains(t, ["Hello", "World"], "World")
//    assert.Contains(t, {"Hello": "World"}, "Hello")
func Contains(t TestingT, s, contains interface{}, msgAndArgs ...interface{}) bool {
```
2. but no matter what the order of the params is - the actuall content of the params is as I said - the array are actual statuses and the second param is the expected one

Reply:
you're right, sorry about this dumb change. Fixed in 2d8cf14